packages/visx-demo/src/sandboxes/visx-geo-albers-usa/yarn.lock

```diff
@@ -0,0 +1,8910 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we only commit the lock file in the root of the monorepo, so if possible can we remove this file?

other examples in `@visx/demo` have the `d3-geo` dependency I believe, so shouldn't need to update the root `yarn.lock` file either. just to explain further: the sandbox examples have a `package.json` so they work on codesandbox. if we were actually adding a new dep here we would need to add it both in the sandbox `package.json` **and** in the `@visx/demo` `package.json`, and then install the dep by running `yarn` in the monorepo root. 

all that to say that if you remove this file, you shouldn't need to do anything else since you added the appropriate dependencies in the sandbox `package.json` 😅